### PR TITLE
Add structlog feedstock.

### PR DESCRIPTION
**Pull request by vamega**

### CODE REVIEW

File: recipes/structlog/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - structlog
+
+about:
+  home: https://github.com/alecthomas/structlog
```
Comment (jakirkham) [inline recipes/structlog/meta.yaml]: This URL does not work.

Comment (jakirkham) [inline recipes/structlog/meta.yaml]: Should it be `https://github.com/hynek/structlog`?

Comment (jakirkham) [inline recipes/structlog/meta.yaml]: According to PyPI it is this `http://www.structlog.org/`. Let's go with this last one.

File: recipes/structlog/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  license: MIT
```
Comment (jakirkham) [inline recipes/structlog/meta.yaml]: Should be `MIT/Apache 2.0`.

File: recipes/structlog/meta.yaml (hunk 1 above)
Comment (vamega) [inline recipes/structlog/meta.yaml]: Yeah, I used a template from another package I was submitting. Forgot to update the URL, will do so.

**Comment by jakirkham** [comment]
Thanks @vamega. LGTM.

**Comment by jakirkham** [comment]
@hynek, just letting you know we have added `structlog` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge structlog`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/structlog-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `structlog`. 😄

**Pull request merged**